src/Data/Time/PreciseDuration.purs

```diff
@@ -52,46 +55,60 @@ unPreciseDuration = case _ of
   Weeks d -> d
 
 -- Each duration in nanoseconds
-nano = BigInt.fromInt 1 :: BigInt
-micro = (nano * BigInt.fromInt 1000) :: BigInt
-milli = (micro * BigInt.fromInt 1000) :: BigInt
-second = (milli * BigInt.fromInt 1000) :: BigInt
-minute = (second * BigInt.fromInt 60) :: BigInt
-hour = (minute * BigInt.fromInt 60) :: BigInt
-day = (hour * BigInt.fromInt 24) :: BigInt
-week = (day * BigInt.fromInt 7) :: BigInt
-
-toNanoseconds' :: PreciseDuration -> BigInt
-toNanoseconds' duration = case duration of
+nano = Decimal.fromInt 1 :: Decimal
```

Comment:
Perhaps this stuff should be hidden in a new module `Data.Time.PreciseDuration.Internal`?

Reply:
I tend to only do that if I need to use internal things across modules, and don't export them otherwise.

I've heard compelling arguments for putting things in internal modules instead though. @natefaubion, what do you think?

Reply:
These things are needed in the tests (and so must be exported), but shouldn't be a part of the public API.

Reply:
In that case, I think an module makes sense.